### ISSUE TITLE
[Bug]: Missing dependencies in ColorField

### ISSUE THREAD

**Issue by Rocss** [adobe/spectrum-web-components#4140]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-color-field

### Expected behavior

`@ctrl/tinycolor` and `@spectrum-web-components/textfield` should be listed as dependencies in package.json

### Actual behavior

The only dependency seems to be `@spectrum-web-components/base`

### Screenshots

N/A

### What browsers are you seeing the problem in?

N/A

### Sample code that illustrates the problem

```
packages/color-field/src/ColorField.ts:20:31 - error TS7016: Could not find a declaration file for module '@spectrum-web-components/textfield'. '........./spectrum-web-components/packages/textfield/src/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/spectrum-web-components__textfield` if it exists or add a new declaration (.d.ts) file containing `declare module '@spectrum-web-components/textfield';`

20 import { TextfieldBase } from '@spectrum-web-components/textfield';
```

**Comment by Rocss** [comment]
@Rajdeepc I think this can be closed, right?

**Comment by Rajdeepc** [comment]
> @Rajdeepc I think this can be closed, right?

Yes this is done!

**Comment by Rajdeepc** [comment]
Fix by #4141
